### PR TITLE
Transfer Travis to Github Actions

### PR DESCRIPTION
**Pull request by Simply007**
### Motivation

Transfer Travis CI to Github Actions

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

If manual testing is required, what are the steps?

### CODE REVIEW

**Comment by Simply007** [comment]
I have removed the special Travis configuration for the CI an test run and turned of single run for default karma config

File: karma.conf.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ module.exports = function (config) {
         port: 9669,
         colors: true,
         autoWatch: true,
-        singleRun: false,
+        singleRun: true,
```
Comment (Enngage) [inline karma.conf.js:48]: @Simply007 can you please bring back the "karma-travis.conf.js"? Maybe rename it like karma-gh.conf.js. I'm using the 'singleRun' only for CI, but for development I want to keep the tests running so I can make dynamic changes to code and have tests run automatically ;)

File: package.json [hunk 2]
```diff
@@ -36,8 +36,7 @@
     "test:node": "npm run build:commonjs && npm run mocha-local -- \"test-node/**/*.js\"",
     "test:browser": "npm run karma-local start karma.conf.js",
     "test:dev": "npm run karma-local start karma-dev.conf.js",
-    "test:travis": "npm run karma-local start karma-travis.conf.js",
-    "test:all": "npm run test:node && npm run test:travis",
+    "test:all": "npm run test:node && npm run test:browser",
```
Comment (Enngage) [inline package.json:39]: Same as above - please add back a script for GH CI such as "test:gh"

**Review by Enngage — CHANGES_REQUESTED** [review]

File: karma.conf.js (hunk 1 above)
Comment (Simply007) [reply karma.conf.js:48]: Ok

File: package.json (hunk 2 above)
Comment (Simply007) [reply package.json]: 77f5c6b

File: karma.conf.js (hunk 1 above)
Comment (Simply007) [reply karma.conf.js]: 77f5c6b

**Comment by Simply007** [comment]
Should be done @Enngage.

**Review by Enngage — APPROVED** [review]
LGTM, thanks!

**Pull request merged**